hudi-common/src/main/java/org/apache/hudi/common/config/HoodieMetadataConfig.java

```diff
@@ -455,6 +456,45 @@ public final class HoodieMetadataConfig extends HoodieConfig {
           + "The index name either starts with or matches exactly can be one of the following: "
           + StringUtils.join(Arrays.stream(MetadataPartitionType.values()).map(MetadataPartitionType::getPartitionPath).collect(Collectors.toList()), ", "));
 
+  // Configs that control the bloom filter that is written to the file footer
+  public static final ConfigProperty<String> BLOOM_FILTER_TYPE = ConfigProperty
+      .key(String.format("%s.%s", METADATA_PREFIX, "bloom.index.filter.type"))
+      .defaultValue(BloomFilterTypeCode.DYNAMIC_V0.name())
+      .withValidValues(BloomFilterTypeCode.SIMPLE.name(), BloomFilterTypeCode.DYNAMIC_V0.name())
+      .markAdvanced()
+      .withDocumentation(BloomFilterTypeCode.class);
+
+  public static final ConfigProperty<String> BLOOM_FILTER_NUM_ENTRIES_VALUE = ConfigProperty
+      .key(String.format("%s.%s", METADATA_PREFIX, "index.bloom.num_entries"))
+      .defaultValue("60000")
```

Comment:
should we reduce this default value. this was chosen based on data files. 
in MDT, each partition could have diff avg no of entries. 
for eg, for FILES, somewhere around 1000. 
for col stats, expression index and bloom index, may be 30000
and for RLI: 100k.